rs/tests/nested/hostos_upgrade.rs

```diff
@@ -119,7 +119,11 @@ pub fn upgrade_hostos(env: TestEnv) {
     .unwrap();
 
     info!(logger, "Waiting for Orchestrator dashboard...");
-    host.await_orchestrator_dashboard_accessible().unwrap();
+    if let Err(e) = host.await_orchestrator_dashboard_accessible() {
+        info!(logger, "Orchestrator dashboard is not accessible: {e}");
+        try_logging_guestos_diagnostics(&host, &logger);
+        panic!("Orchestrator dashboard is not accessible: {e}");
```

Comment:
nit: I don't know that we need both with the same log, so we could either change the panic text, or drop the info log.